engine/src/main/java/org/camunda/bpm/engine/impl/cfg/ProcessEngineConfigurationImpl.java

```diff
@@ -954,6 +954,15 @@ public abstract class ProcessEngineConfigurationImpl extends ProcessEngineConfig
    */
   protected int historyCleanupDefaultNumberOfRetries = Integer.MIN_VALUE;
 
+  /**
+   * When set to false, exclusivity (no parallel execution) of tasks is applied per process instance.
+   * When set to true, exclusivity (no parallel execution) of tasks is extended across all process instances.
+   * <p>
+   * Default value: false; to keep the legacy behaviour backwards compatible.
+   * //TODO add rest of javadoc, mention performance implications.
+   */
+  protected boolean enableExclusivenessAcrossProcessInstances = false;
```

Comment:
❓ Can we reflect in the property's name this feature is about job acquisition and process hierarchies? Can we reuse the naming scheme we already have for job executor settings? I can see that we already use the `jobExecutorAcquire*` prefix. Maybe something like `jobExecutorAcquireExclusiveOverProcessHierarchies`?

Reply:
Renamed to the suggested example